README.md

```diff
@@ -13,8 +13,22 @@ Create a Python 3.6 virtual environment and install the packages:
     virtualenv venv
     source venv/bin/activate
     pip install -r requirements.txt
-
+    pip install -e .
+ 
 ```
 Run
 ./app.py
-```
\ No newline at end of file
+```
+
+
+
+
+Contributing
+------------
+
+Really? Very welcome. Do the usual fork-and-submit-PR thingy.
+
+Running the tests:
+
+    python setup.py test
```

Comment:
Have you tried `deactivate; rm -rf venv`, running the *Getting Started* steps from scratch, then this passes the 2 dead-simple tests recommended in PR #1 comment..? Can't try it out myself atm.